doc/changes/latest.inc

```diff
@@ -113,6 +113,8 @@ Enhancements
 
 - Add :func:`mne.viz.Brain.add_volume_labels` to plot subcortical surfaces and other regions of interest (:gh:`9540` by `Alex Rockhill`_ and `Eric Larson`_)
 
+- Add split :func:`mne.viz.Brain.show_view` argument ``view`` into ``azimuth``, ``elevation`` and ``focalpoint`` for clearer view setting (:gh:`9596` by `Alex Rockhill`_)
```

Comment:
This should actually go under API because it introduces a deprecation

Reply:
... also there are more deprecations than this, i.e., `row` and `col` both need deprecation cycles